src/validate_pyproject/vendoring/cli.py

```diff
@@ -1,10 +1,12 @@
+from typing import Any
+
 from ..pre_compile import cli
 from . import _deprecated
 
 
-def run(*args, **kwargs):
+def run(*args: Any, **kwargs: Any) -> Any:
```

Comment:
I assume this won't be here forever, so didn't bother to work out the details exactly. In the future I'd assume we'd use `typing_extensions.deprecated` / `warnings.deprecated` (3.13+) instead.

Reply:
We can probably drop this soon.
I don't imagine there is someone using this part of the API.